src/placeos-models/user.cr

```diff
@@ -28,6 +28,8 @@ module PlaceOS::Model
     attribute first_name : String?
     attribute last_name : String?
     attribute building : String?
+    attribute department : String?
+    attribute preferred_language : String?
```

Comment:
What's the form of this? We can add some validation to ensure it's a valid language code